sources/packages/backend/libs/test-utils/src/factories/disbursement-receipt.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  DisbursementReceipt,
+  DisbursementSchedule,
+  InstitutionLocation,
+  Student,
+} from "@sims/sims-db";
+import { getISODateOnlyString } from "@sims/utilities";
+import * as faker from "faker";
+
+/**
+ * Create fake disbursement receipts.
+ * @param relations dependencies.
+ * - `disbursementSchedule` related disbursement schedule.
+ * - `student` related student.
+ * - `institutionLocation` related institution location.
+ * @returns disbursement receipt.
+ */
+export function createFakeDisbursementReceipt(relations: {
+  disbursementSchedule: DisbursementSchedule;
```

Comment:
`disbursementSchedule.studentAssessment.application.student` can be used instead of receiving student as argument.

Reply:
actually, I prefer to pass a relation as a relation variable